spec/models/user_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :each do
+    @user = User.new(name: 'John Doe', photo: 'https://example.com/photo.jpg', bio: 'hola', posts_counter: 0)
+    @user.save
+  end
+
+  it 'name should be present' do
+    @user.name = nil
+    expect(@user).to_not be_valid
+  end
+
+  it 'photo always valid' do
```

Comment:
- [ ] Your Model specs should have at least one **Validation** test. Kindly, make sure that you implement it. Take a look at this example below: 
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

Reply:
HI, please can you show me that what you are suggesting works?

I mean I have 24 validations made and you said that there is no one?

let me explain to you how I have done this:
![Screenshot from 2022-07-07 13-19-32](https://user-images.githubusercontent.com/96317674/177842523-ba4537f4-2555-4aa1-b421-2ffb2594cf69.png)

in Line 10 in my model, I'm passing PRESENCE: TRUE

so there is no need to modify it to pass validates_presence_of: 

and then in my tests in line 11 as you can see I'm validating presence of 'name' so if 'name' is nil the '@user should not be valid'

![Screenshot from 2022-07-07 13-28-53](https://user-images.githubusercontent.com/96317674/177844043-5ffea951-400e-4af4-9c50-01819456a773.png)